src/components/Setup/index.tsx

```diff
@@ -293,8 +323,7 @@ export default class Setup extends React.Component<any, ISetupState> {
               />
             </div>)
         }
-        <Button flat primary label="Save &amp; Apply" onClick={this.onSave}>save</Button>
```

Comment:
Also if I switch Auth off and reopen the setup page this preference isn't saved.